solutions/cf-920E.mdx

```diff
@@ -0,0 +1,233 @@
+---
+id: cf-920E
+source: CF
+title: Connected Components?
+author: Daniel Qiu
+---
+
+
+[Official Editorial](https://codeforces.com/blog/entry/57516)
+
+
+## Solution 1 - Faster DFS
+
+Doing a standard DFS would be too slow. To speed it up, we have to do two
+optimizations.
+
+### Optimization 1
+
+We can store whether non-existant edges using an array of hashsets, instead of a adjacency list. This way we can find if there is no edge between two nodes in
```

Comment:
```suggestion
We can store whether edges are non-existent using an array of hashsets instead of an adjacency list. This way we can find if there is no edge between two nodes in
```